spec/02-integration/05-proxy/17-origins_spec.lua

```diff
@@ -16,7 +16,7 @@ describe("origins config option", function()
       proxy_client:close()
     end
 
-    helpers.stop_kong(nil, true)
+    helpers.stop_kong()
```

Comment:
The "do not preserve prefix" commit needs to be merged with its parent.

Reply:
The commit that introduced this issue isn't in the current PR.